src/main/java/com/openlattice/hazelcast/HazelcastMap.kt

```diff
@@ -101,40 +100,38 @@ class HazelcastMap<K, V> internal constructor(val name: String) : TypedMapIdenti
         @JvmField val ENTITY_TYPE_PROPERTY_METADATA = HazelcastMap<EntityTypePropertyKey, EntityTypePropertyMetadata>("ENTITY_TYPE_PROPERTY_METADATA")
         @JvmField val ENTITY_TYPES = HazelcastMap<UUID, EntityType>("ENTITY_TYPES")
         @JvmField val EXPIRATION_LOCKS = HazelcastMap<UUID, Long>("EXPIRATION_LOCKS")
+        // todo is this used?
```

Comment:
It should be, unless someone got rid of it. It's used for managing permissions for organization database tables/columns for the stuff we're doing for the state of california.

Reply:
Do you remember what the Mapstore class was called? I added the todo here, bc the ide is showing me, that it's not used :(

Reply:
Hmmm looking back through my PR I think I actually just pull directly from postgres, so I may have added the map without ever making a mapstore and forgotten to remove it :|. Feel free to delete.

Reply:
Thanks! Will do!